src/devices/SDLJoypad/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ yarp_prepare_plugin(SDLJoypad
 if(ENABLE_SDLJoypad)
 
   set(CMAKE_INCLUDE_CURRENT_DIR ON)
+  set (CMAKE_CXX_STANDARD 11)
```

Comment:
Actually it seems to Be required by some ide to autocomplete standard c++11 specific method (e.g.. std::toString()).. or maybe there are some trick I don't know.. in case, let me know